src/helm/benchmark/contamination/export_scenario_text.py

```diff
@@ -25,8 +25,8 @@ def get_light_scenarios_from_run_spec(
     run_spec: RunSpec, scenario_download_path: str = "exported_scenarios"
 ) -> List[LightScenario]:
     """
-    Create a list of LightInstances given a RunSpec. Only keep the text attributes. Note that one LightScenario
-    object is created for each split of the Scenario for simplification.
+    Create a list of LightInstances given a RunSpec. Only keep the text of the input and references.
+    Note that one LightScenario object is created for each split of the Scenario for simplification.
```

Comment:
Honestly this comment makes me think that the class should be named `ScenarioSplitContamination` or something. No change needed currently, but we should consider renaming this in the future.